src/sql/parts/modelComponents/modelViewContent.component.ts

```diff
@@ -26,7 +26,7 @@ import * as sqlops from 'sqlops';
 	selector: 'modelview-content',
 	template: `
 		<div *ngIf="rootDescriptor" style="width: 100%; height: 100%;">
-			<model-component-wrapper [descriptor]="rootDescriptor" [modelStore]="modelStore">
+		<model-component-wrapper style="display: block; height: 100%" [descriptor]="rootDescriptor" [modelStore]="modelStore">
```

Comment:
Tabbing got messed up. Better to set the style via css if possible but it's not that important